src/api/stakeholders.js

```diff
@@ -0,0 +1,189 @@
+import { handleResponse, handleError, getOrganizationId } from './apiUtils';
+import { session } from '../models/auth';
+const log = require('loglevel').getLogger('../api/stakeholders');
+
+const STAKEHOLDER_API = process.env.REACT_APP_STAKEHOLDER_API_ROOT;
+
+async function fetchJSON(query, options) {
+  return fetch(query, options).then(handleResponse).catch(handleError);
+}
+
+export default {
+  getStakeholders(id, { offset, rowsPerPage, orderBy, order, filter }) {
+    const orgId = id || getOrganizationId();
+    const filterObj = {
+      where: filter,
+      order: [`${orderBy} ${order}`],
+      limit: rowsPerPage,
+      offset,
+    };
+
+    let query = `${STAKEHOLDER_API}?filter=${JSON.stringify(filterObj)}`;
+
+    if (orgId) {
+      query = `${STAKEHOLDER_API}/${orgId}?filter=${JSON.stringify(filterObj)}`;
+    }
+
+    log.debug('getStakeholders', query);
+
+    const options = {
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json',
+        Authorization: session.token,
+      },
+    };
+
+    return fetchJSON(query, options);
+  },
+
+  async getUnlinkedStakeholders(id, abortController) {
+    const orgId = getOrganizationId();
+    let query = `${STAKEHOLDER_API}/relations`;
```

Comment:
The `/relations` endpoint of stakeholder API returns a 500 error when clicking on a stakeholder - `column "owner_id" does not exist`. Maybe this is because the deployed version of the API not the latest?

Reply:
This is due to an update to the API and database that removed the owner_id column

Reply:
This is because of an update to the API and the removal of the owner_id column.  We'll need to figure out if it's needed or not and the update.